src/Files.App/Contexts/DisplayPage/DisplayPageContext.cs

```diff
@@ -100,6 +93,17 @@ public SortDirection GroupDirection
 			}
 		}
 
+		private GroupByDateUnit groupByDateUnit = GroupByDateUnit.Year;
+		public GroupByDateUnit GroupByDateUnit
```

Comment:
groupByDateUnit -> _GroupByDateUnit

According to the guideline, section 2, item 2
> Use PascalCase prefixed with _ for properties’ backing fields